spec/javascripts/tests/Header_spec.js

```diff
@@ -0,0 +1,65 @@
+describe('Header', function() {
+  'use strict';
+
+  beforeEach(function(done) {
+    var self = this;
+
+    fixture.setBase('spec/javascripts/fixtures');
+
+    requirejs(
+      ['jquery', 'Header'],
+      function($, Header) {
+        fixture.load('Header.html');
+
+        self.$header = $(fixture.el).find('[data-dough-component="Header"]');
+        self.obj = new Header(self.$header);
+
+
+        self.activeClass = 'mobile-header__search-active';
```

Comment:
Better to get the name of the active class from Header.js as: self.obj.activeClass (in case the class name changes in the component).

Reply:
@davidtrussler as discussed, leaving this change or it'll make the test obsolete